src/Compilers/Core/MSBuildTask/DiscoverEditorConfigFiles.cs

```diff
@@ -24,7 +24,12 @@ public sealed class LocateEditorConfigFiles : Task
         [Output]
         public ITaskItem[] EditorConfigFiles { get; private set; }
 
-        public LocateEditorConfigFiles()
+        static DiscoverEditorConfigFiles()
+        {
+            AssemblyResolution.Install();
```

Comment:
(which I guess is potentially being cleaned up by your other PR doing the downgrade, but until then yes this is needed)